definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -6298,5 +6298,61 @@
     "CaseEventID": "amendPetitionForRefusalRejection",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
```

Comment:
`caseworker-divorce-systemupdate` and `caseworker-divorce-bulkscan` should be able to CRU on `createCase` and `updateCase` (new events - names defined by RBS)